python-lib/dku_timeseries/resampling.py

```diff
@@ -83,13 +83,19 @@ def transform(self, df, datetime_column, groupby_columns=None):
         if groupby_columns:
             grouped = df_copy.groupby(groupby_columns)
             resampled_groups = []
+            identifiers_number = len(groupby_columns)
             for group_id, group in grouped:
                 logger.info("Computing for group: {}".format(group_id))
                 group_resampled = self._resample(group.drop(groupby_columns, axis=1), datetime_column, columns_to_resample, reference_time_index,
                                                  df_id=group_id)
-                group_resampled.loc[:, groupby_columns[0]] = group_id  # TODO make this work with multiple group cols
+                if identifiers_number == 1:
+                    group_id = [group_id]
+                else:
+                    group_id = list(group_id)
+                group_resampled[groupby_columns] = pd.DataFrame([group_id], index=group_resampled.index)
                 resampled_groups.append(group_resampled)
-            df_resampled = pd.concat(resampled_groups)
+                logger.info("Computing for group: {} - Done!".format(group_id))
```

Comment:
No need for this line, it doubles the number of log lines that can already be big when there are many time series